packages/KDateTimePicker/KDateTimePicker.vue

```diff
@@ -0,0 +1,502 @@
+<template>
+  <KPop
+    :hide-popover="hidePopover"
+    class="k-datetime-picker"
+    placement="bottomStart"
+    width="auto"
+    hide-caret
+    position-fixed
+    @closed="handleClose"
+  >
+    <KButton
+      :is-rounded="false"
+      size="large"
+      class="timepicker-input"
+    >
+      {{ abbreviatedDisplay }}
+    </KButton>
+    <template
+      v-if="!hidePopover"
+      #content>
+      <KSegmentedControl
+        v-if="hasRelativeTimeframes && hasCalendar"
+        v-model="tabName"
+        :options="[
+          { label: 'Relative', value: 'relative' },
+          { label: 'Custom', value: 'custom' }
+        ]"
+        class="w-100 mb-5"
+        data-testid="analytics-time-toggle"
```

Comment:
We don't want to reference `konnect` or `analytics` in any of the components
```suggestion
        data-testid="k-datetime-picker-time-toggle"
```